content/docs/capabilities/tcp/client.mdx

```diff
@@ -49,116 +60,59 @@ The example below demonstrates a route to the SSH service on the host running th
               is: user@companydomain.com
 ```
 
-</TabItem>
-<TabItem value="Pomerium Enterprise" label="Pomerium Enterprise">
-
-![Example TCP route for SSH](./img/tcp-ssh-route.png)
-
 </TabItem>
 </Tabs>
 
-See the "Configure Routes" section of [TCP Support](/docs/capabilities/tcp#configure-routes) for more detailed information on TCP routes.
-
-## TCP Client Software
-
-You can connect to this route with either the Pomerium CLI or Pomerium Desktop client.
-
-<Tabs>
-
-<TabItem value="Pomerium Desktop" label="Pomerium Desktop">
-
-### Install
-
-Download the latest release from [GitHub](https://github.com/pomerium/desktop-client/releases).
-
-- **Windows**: The installer `.exe` file will install and open the Desktop Client. Right click on the system tray icon to interact with it.
-- **Linux**: We provide Linux binaries as `.AppImage` files, which can be executed in place or managed with a tool like [AppImageLauncher](https://github.com/TheAssassin/AppImageLauncher). Interact with the client from the system tray icon.
-- **macOS**: Open the `dmg` and move the binary to **Applications**. Interact with the client from the system tray icon.
-
-<details>
-<summary>Autostart Pomerium Desktop</summary>
-
-If you want Pomerium Desktop to start automatically when you log in to your computer, follow the steps below for your operating system.
-
-<Tabs>
-<TabItem value="windows" label="Windows">
-
-#### Autostart for all users
+:::note
 
-Copy the shortcut for the Pomerium Desktop app into `C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Startup`.
+The port suffix appended to the **From** route is not used to connect to the Pomerium Proxy service from the internet; this will always be port 443 (unless otherwise defined in the configuration file).
```

Comment:
Sorry, I think I deleted a pending comment somehow. I think this could be made a little clearer, along the lines of:

```markdown
:::tip

The port number in the route **From** URL is not used in the initial connection
to Pomerium itself. This connection will still use port 443, unless you use a
bastion host (see [Advanced configuration](#advanced-configuration) below).
```